### PR TITLE
[BuildRegistryModule]:After adding fields in common schema - Only Fie…

### PR DESCRIPTION
**Pull request by suvarnakale**
…lds Menu showing up- Issue Fixed

### CODE REVIEW

File: src/app/admin/create-entity/create-entity.component.html [hunk 1]
```diff
@@ -118,38 +118,36 @@ <h3>{{usecaseSchema[activeMenuNo]?.title}}</h3>
...
                                                             {{secArrayfield?.propertyName}}
-
+                                                            
                                                         </p>
                                                         <ng-template ngFor let-childNastedArrayfield
                                                             [ngForOf]="secArrayfield?.data" let-m="index">
 
                                                             <tr [hidden]="!childNastedArrayfield?.data?.title"
-                                                                style="border: white solid;">
+                                                                style="border: white solid;" >
```
Comment (Pratikshakhandagale) [inline src/app/admin/create-entity/create-entity.component.html:150]: @suvarnakale remove this inline css

File: src/app/admin/create-entity/create-entity.component.html [hunk 2]
```diff
@@ -199,7 +197,7 @@ <h3>{{usecaseSchema[activeMenuNo]?.title}}</h3>
                                                     </ng-template>
 
                                                     <tr [hidden]="!secArrayfield?.data?.title"
-                                                        style="border: white solid;">
+                                                        style="border: white solid;" class="border-bottom">
```
Comment (Pratikshakhandagale) [inline src/app/admin/create-entity/create-entity.component.html:200]: @suvarnakale remove this inline css

File: src/app/admin/create-entity/create-entity.component.html [hunk 3]
```diff
@@ -243,7 +241,7 @@ <h3>{{usecaseSchema[activeMenuNo]?.title}}</h3>
 
                                                 </ng-template>
 
-                                                <tr style="border: white solid;"  [hidden]="!field[j]?.title">
+                                                <tr style="border: white solid;"  [hidden]="!field[j]?.title" >
```
Comment (Pratikshakhandagale) [inline src/app/admin/create-entity/create-entity.component.html:244]: @suvarnakale remove this inline css

File: src/app/admin/create-entity/create-entity.component.html (hunk 2 above)
Comment (suvarnakale) [reply src/app/admin/create-entity/create-entity.component.html]: Done

**Pull request merged**